natgeo/mortar
# [SYTO-884] Style Video Modal

https://jira.natgeo.com/browse/SYTO-884

This ticket implement the Video Modal on mortar.
There is a refactor on the Modal Component to allow children and render them inside the modal content.

#### How to test
1. ``gulp serve``
2. Open /components
3. Scroll down until the Video Modal section
4. Click on "Launch Modal"
5. See the video with the proper information.
